includes/functions-ur-template.php

```diff
@@ -88,7 +88,9 @@ function ur_registration_template_redirect() {
 
 		if ( has_shortcode( $post_content, 'user_registration_form' ) ) {
 
-			$redirect_url = get_option( 'user_registration_general_setting_redirect_options' );
+			$attributes   = ur_get_shortcode_attr( $post_content );
+			$form_id 	  = isset( $attributes[0]['id'] ) ? absint( $attributes[0]['id'] ) : 0;
```

Comment:
@rumess Form ID is converted to 0 with absint() due to the id constructed in string. For example: ""60"". Make sure the form id is correct before merging the PR.

Reply:
I have used prag_match_all to strip integer from a string value.